docs/app/Examples/collections/Form/Shorthand/FormExampleSubComponentControl.js

```diff
@@ -16,8 +16,8 @@ class FormExampleSubComponentControl extends Component {
     return (
       <Form>
         <Form.Group widths='equal'>
-          <Form.Input label='First name' placeholder='First name' />
-          <Form.Input label='Last name' placeholder='Last name' />
+          <Form.Input id='form-input-shorthand-first-name' label='First name' placeholder='First name' />
+          <Form.Input id='form-input-shorthand-last-name' label='Last name' placeholder='Last name' />
```

Comment:
I'd like to have two dedicated examples for this in the docs under one heading.  One example showing the usage with `Form.Field` and another showing usage with `Form` shorthand subcomponents.  

First, create two example files in `docs/app/Examples/collections/Form/Shorthand`, probably:
1. `FormExampleFieldControlId.js` showing `Form.Field` use with the shorthand `control` prop.
2. `FormExampleSubcomponentId.js` showing usage of one or more `Form` subcomponents, i.e. `Form.Input`.

Then, add these files to the `index.js` file in the same directory.  You link examples with the `ComponentExample` component, you'll see several in the `index.js` file.  In order to get two examples to show under one title, only add a `title` to the first example but `descriptions` to both:

```jsx
<ComponentExample
  title='Accessible lables'
  description='Adding an id to a shorthand Form.Field adds a matching htmlFor prop to the label.'
  examplePath='collections/Form/Shorthand/FormExampleFieldControlId'
/>
<ComponentExample
  description='Adding an id to a Form subcomponent adds a matching htmlFor prop to the label.'
  examplePath='collections/Form/Shorthand/FormExampleSubcomponentId'
/>
```